### ISSUE TITLE
Preserve annotation descriptions when converting to events

### ISSUE THREAD

**Issue by cbrnr** [cbrnr/mnelab#167]
Currently, we lose annotation descriptions when converting annotations to events. However, `mne.events_from_annotations` returns a tuple with the first element containing the event array and the second element containing the mapping from annotation description to event ID. I suggest that we store this tuple in `model.current["events"]` instead of only the event array. In cases where this mapping is not available, we could still use a 1-tuple and only store the event array.

**Comment by abcsds** [comment]
Saving the event mappings as a tupple on `model.current["events"]` would require a change on every method that uses the current `model.current["events"]`. Wouldn't it be better to create a new variable for example `model.current["event_mappings"]` and use it when converting from events to annotations. This is a simpler fix. We don't really have any other use for the mappings apart from converting back to annotations, do we?

**Comment by cbrnr** [comment]
It's only used in 5 places, so these changes would be doable. But I agree your suggestion is a good alternative and doesn't require any changes so we might as well go for that.

**Comment by abcsds** [comment]
Here is my proposal:
I'm just checking if the variable has been created before changing back to annotations.
To write the history I used a multi-line string. I did this to keep the indentation of the if statement, but I just noticed that you do the check for the variable outside the history, and keep the history one line (I saw it in the `plot_data` function of the main window. I'm changing my PR to that.
